website/src/docs/transloadit.md

```diff
@@ -9,40 +9,40 @@ The Transloadit plugin can be used to upload files to [Transloadit](https://tran
 
 [Try it live](/examples/transloadit/)
 
-The Transloadit plugin uses the [Tus plugin](/docs/tus) for the uploading itself.
-To upload files to Transloadit directly, both the Tus and Transloadit plugins must be used:
-
 ```js
-// The `resume: false` option _must_ be provided to the Tus plugin.
-uppy.use(Tus, {
-  resume: false
-})
 uppy.use(Transloadit, {
-  // Transloadit plugin options
+  service: 'https://api2.transloadit.com',
+  waitForEncoding: false,
+  waitForMetadata: false,
+  importFromUploadURLs: false,
+  alwaysRunAssembly: false,
+  params: null,
+  signature: null,
+  fields: {}
 })
 ```
 
-NB: It is not required to use the `Tus` plugin if [importFromUploadURLs](#importFromUploadURLs) is enabled.
+As of Uppy 0.24 the Transloadit plugin includes the [Tus](/docs/tus) plugin to handle the uploading, so you no longer have to add it manually.
```

Comment:
Writing the version number here because this PR will probably be merged before we release the change, and it would be confusing otherwise … versioned docs would be rad, but probably a lot of work!